Makefile

```diff
@@ -1,3 +1,36 @@
+TARGET_GOOS=linux
+TARGET_GOARCH=amd64
+
+# Export GO111MODULE=on to enable project to be built from within GOPATH/src
+export GO111MODULE=on
+# Enable GOPROXY. This speeds up a lot of vendoring operations.
+export GOPROXY=https://proxy.golang.org
+# Export GOROOT. Required for OPERATOR_SDK to work correctly for generate commands.
+export GOROOT=$(shell go env GOROOT)
+
+.PHONY: \
+	clean \
+	check-scripts \
+	functest \
+	govet \
+	unit-test
+
+clean:
+	@echo "cleaning previous outputs"
+	hack/clean.sh
+
+build-functest:
+	@echo "Building functional tests"
+	hack/build-functest.sh
+
+functest: build-functest
+	@echo "Running functional test suite"
+	hack/functest.sh
+
+govet:
```

Comment:
govet target doesn't return exit code 0 atm. Although it would be a nice extra to add once it is working, I would omit it here because it's not related to functional tests.

Reply:
:+1: